### PR TITLE
Move color-dot option to labels

### PR DESCRIPTION
**Pull request by Nitwel**
Adds the show-as-dot option to labels.
Color interface now only accepts hex values.

grafik

I had to change the type validator on labels to `text, string` from `json, string` cause json gets changed to text.
I'm not sure if this a intended behaviour.

### CODE REVIEW

**Comment by benhaynes** [comment]
Love this... and it's really nice that this supports multiple values now too!

**Comment by benhaynes** [comment]
Do we have some smart logic for when the dot color is white (or `$page_background`) to outline it with a light gray border for visibility?

**Comment by Nitwel** [comment]
Not yet, but thats a good edgecase. Would it be enough to check if the color is the same as `--background-page` or should we validate against a contrast standard?

**Comment by benhaynes** [comment]
Contrast would be ideal — and we have other places that will be useful too (eg: the `color` interface)

**Comment by rijkvanzanten** [comment]
@Nitwel can you get this branch up to date with main?

**Comment by Nitwel** [comment]
Contrast now gets calculated relative to `--background-page` css var.

File: api/src/cli/utils/drivers.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const drivers = {
 	sqlite3: 'SQLite',
-	mysql: 'MySQL (/ MariaDB / Aurora)',
-	pg: 'PostgreSQL (/ Amazon Redshift)',
+	mysql: 'MySQL / MariaDB / Aurora',
+	pg: 'PostgreSQL / Redshift',
```
[the remark this answers is not in the document]
Comment (rijkvanzanten) [reply api/src/cli/utils/drivers.ts:4]: Sneak in an unrelated change on the show-as-dot branch

[the remark this answers is not in the document]
Comment (benhaynes) [reply api/src/cli/utils/drivers.ts:4]: Haha, must have been merged upstream, or whatever... I only push to `main` baby!

File: app/package.json [hunk 2]
```diff
@@ -45,6 +47,8 @@
     "base-64": "^0.1.0",
     "bytes": "^3.1.0",
     "codemirror": "^5.56.0",
+    "color": "^3.1.2",
+    "color-string": "^1.5.3",
```
Comment (rijkvanzanten) [inline app/package.json:51]: Do we need both `color` (22.4kB) and `color-string` (6.8kB) ?

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**